### PR TITLE
Various minor bugfixes

### PR DESCRIPTION
**Pull request by ChrisKitching**

### CODE REVIEW

File: dist/screeps.d.ts [hunk 1]
```diff
@@ -1102,7 +1102,7 @@ interface FindPathOpts {
      */
     maxRooms?: number;
 }
-interface MoveToOpts {
+interface MoveToOpts extends FindPathOpts {
```
Comment (Dessix) [inline dist/screeps.d.ts:878]: Acceptable, assuming there are no places without FindPathOpts included in MoveToOpts usage.

**Review by Dessix — APPROVED** [review]

**Review by Dessix — CHANGES_REQUESTED** [review]
Approved pending conflict resolution and checking of changes involved.

File: dist/screeps.d.ts (hunk 1 above)
Comment (ChrisKitching) [reply dist/screeps.d.ts:878]: This formulation matches the manual:
http://support.screeps.com/hc/en-us/articles/203013212-Creep#moveTo

Under the definition of of `opts`, it explicitly states:
> Any options supported by Room.findPath method.

So: any code which is relying on MoveToOpts not containing members from FindPathOpts is wrong.

Comment (Dessix) [reply dist/screeps.d.ts:878]: It has been merged, so thank you 👍

**Pull request merged**